api/server/server.go

```diff
@@ -44,7 +43,8 @@ type Server struct {
 type Config struct {
 	LotusAddress    ma.Multiaddr
 	LotusAuthToken  string
-	GrpcHostAddress ma.Multiaddr
+	GrpcHostNetwork string
+	GrpcHostAddress string
```

Comment:
Just decided to make this configurable because you can actually set it up to use uds or other things for testing etc